src/variables/Variable.php

```diff
@@ -34,6 +34,46 @@ public function lists(): mixed
         return $this->lists;
     }
 
+    /**
+     * profileFromList.
+     *
+     * @author	Unknown
+     * @since	v0.0.1
+     * @version	v1.0.0	Monday, November 14th, 2022.
+     * @access	public
+     * @param	mixed	$listId	
+     * @param	mixed	$email 	
+     * @return	object
+     */
+    public function profileFromList($listId, $email)
+    {
+        if($listId && $email) {
+            $profile = Plugin::getInstance()->api->getProfileFromList($listId, $email);
+        }
+
+        return $profile;
+    }
+
+    /**
+     * profileFromList.
+     *
+     * @author	Unknown
+     * @since	v0.0.1
+     * @version	v1.0.0	Monday, November 14th, 2022.
+     * @access	public
+     * @param	string	$profileId	
+     * @return	array
+     */
+    public function profile($profileId)
```

Comment:
Need to include Typing on this method. Indicate what type of thing the params are, and the type of thing that is being returned.